src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -234,7 +234,24 @@ private void ValidateLinuxSKUConfiguration()
                 _environment.IsLinuxConsumptionOnAtlas() &&
                 !_environment.IsManagedAppEnvironment())
             {
-                _logger.LogError($"Unable to load the functions payload since the app was not provisioned with valid {AzureWebJobsSecretStorage} connection string.");
+                logger?.LogError($"Unable to load the functions payload since the app was not provisioned with valid {AzureWebJobsSecretStorage} connection string.");
```

Comment:
Nice. This has the added benefit that this message will now go to customers rather than just to us.